doc/changelog.rst

```diff
@@ -11,56 +11,125 @@ Jump to :ref:`SmartRedis Changelog <sr_changelog>`
 SmartSim
 ========
 
+0.6.1
+-----
 
-Development branch
-------------------
-
-To be released at some future point in time
+Released on 15 February, 2024
 
 Description
 
-- Updated tests to address Torch warning
-- Updated GitHub actions to latest versions in CI/CD
-- Dropped Cobalt support
-- Override the sphinx-tabs extension background color
-- Updated SmartSim's machine learning backends
-- Added ONNX support for Python 3.10
-- Added support for Python 3.11
-- Added support for SmartSim with Torch on Apple Silicon
-- Updated install instructions for Apple Silicon
+- Fix line-ending related errors on MacOS (ARM64)
+- Update instructions for Apple Silicon
+- Duplicate DBModel/Script prevention
+- Update license to include 2024
+- Patch changes for release prep
+- Manifest: has DB objects refactor
+- Add support for Mac OSX on Apple Silicon
+- Add eval() to remove Torch warnings during testing
+- Validate Slurm timing format
+- Expose Typehints
+- Fix test_logs to prevent generation of dir
+- Fix Python Typehint for RunSettings.colocated_db_settings
+- Add isort/black check to github actions
+- Relax typing extensions required version
+- Python 3.11 Support
+- Quality of life `smart validate` improvements
+- Update outdated actions
```

Comment:
This can probably be dropped from the short description just because it is strictly "our" CI that changed